File: FitTrack-Backend/FitTrack.Data.Contract/Helpers/Responses/AuthenticationResponse.cs
using FitTrack.Data.Object.Enums;

namespace FitTrack.Data.Contract.Helpers.Responses;

public class AuthenticationResponse
{
    public required string FirstName { get; set; }
    public required string LastName { get; set; }
    public required List<string> Permissions { get; set; }
    public UnitSystemEnum UnitSystem { get; set; }
    public AppThemeEnum AppTheme { get; set; }
}

File: FitTrack-Frontend/src/app/responses/authentication/authentication.response.ts
import { AppThemeEnum } from '../../enums/app-theme.enum';
import { UnitSystemEnum } from '../../enums/unit-system.enum';

export interface AuthenticationResponse {
  firstName: string;
  lastName: string;
  permissions: string[];
  unitSystem: UnitSystemEnum;
  appTheme: AppThemeEnum;
}
